__docs__/wonder-blocks-dropdown/multi-select.argtypes.ts

```diff
@@ -17,15 +29,15 @@ const argTypes: ArgTypes = {
     labels: {
         control: {type: "object"},
         description:
-            "The object containing the custom labels used inside this component.",
+            "The object containing the custom values used inside this component.",
```

Comment:
Same thing from the singleselect can be applied to the multiselect docs!
```suggestion
            "The object containing the custom label values used inside this component.",
```